### ISSUE TITLE
flocking model example does not work

### ISSUE THREAD

**Issue by lmiq** [JuliaDynamics/Agents.jl#951]
The flocking model example is not working, because the `Polygon` type apparently is not available anymore in Makie:

```julia
julia> const bird_polygon = Makie.Polygon(Point2f[(-1, -1), (2, 0), (-1, 1)])
ERROR: UndefVarError: `Polygon` not defined
Stacktrace:
 [1] getproperty(x::Module, f::Symbol)
   @ Base ./Base.jl:31
 [2] top-level scope
   @ REPL[16]:1

(jl_rogOgj) pkg> st
Status `/tmp/jl_rogOgj/Project.toml`
  [46ada45e] Agents v5.17.1
  [13f3f980] CairoMakie v0.11.4
  [ee78f7c6] Makie v0.20.3

julia> Makie.Polygon
ERROR: UndefVarError: `Polygon` not defined
Stacktrace:
 [1] getproperty(x::Module, f::Symbol)
   @ Base ./Base.jl:31
 [2] top-level scope
   @ REPL[21]:1


```

**Comment by Tortar** [comment]
mmmh, It seems like that pinning the version of Makie should be the right choice to avoid these problems in the future

**Comment by Tortar** [comment]
Adding to v6.0 because we will update anyway to Makie 0.20 (even if the released version now includes the previous version of Makie so flocking should work)

**Comment by fbanning** [comment]
This issue should be fixed with #934
